packages/apriltag/src/apriltag_postprocessing_node.py

```diff
@@ -149,10 +149,10 @@ def callback(self, msg):
             tagzout_T_tagxout = tr.euler_matrix(-np.pi/2, 0, np.pi/2, 'rxyz')
 
             #Load translation
-            trans = detection.pose.pose.pose.position
-            rot = detection.pose.pose.pose.orientation
-            header = detection.pose.header
-
+            trans = detection.transform.translation
+            rot = detection.transform.rotation
+            header = Header()
+            header.stamp = rospy.Time.now()
```

Comment:
does the incoming message not have a hearder.stamp? normally we carry the stamps through so that we can calculate total latency.